starfish/test/full_pipelines/api/test_dartfish.py

```diff
@@ -1,26 +1,24 @@
+import os
+import sys
+
 import numpy as np
 import pandas as pd
+from definitions import ROOT_DIR
 
-from starfish import Experiment, IntensityTable
-from starfish.image._filter.scale_by_percentile import ScaleByPercentile
-from starfish.image._filter.zero_by_channel_magnitude import ZeroByChannelMagnitude
-from starfish.spots._detector.pixel_spot_detector import PixelSpotDetector
+from starfish import IntensityTable
 from starfish.types import Features
 
+os.environ["USE_TEST_DATA"] = "1"
+sys.path.append(os.path.join(ROOT_DIR, "notebooks/py/"))
```

Comment:
this should be `os.path.join(ROOT_DIR, "notebooks", "py")` in case someone is on Windows.